docs/deploy-development-rp.md

```diff
@@ -119,9 +119,6 @@
    1. use the create utility:
 
       ```bash
-      # Create the application to run the cluster as and load it
-      CLUSTER=<cluster-name> go run ./hack/cluster createapp
```

Comment:
Do we want to keep providing a custom SP in local dev as an option instead of forcibly auto-creating one?

We could still use `az` with dev extension, although we would have to create it in a few more steps, we wouldn't use the smaller workers SKU and it would be easier to forget to delete the app (there would be no file around to remind us we have a lingering app/SP)

Reply:
Talked to Maitiu and we are carding it as a follow-up to allow to merge this PR as it is, since we want to get the change reverted quickly and this card can be also a good-first-item:
https://issues.redhat.com/browse/ARO-9320